lib/apple/index.js

```diff
@@ -32,18 +46,27 @@ function parseResult(result) {
 		throw error;
 	}
 
-	var receipt = result.receipt;
+	var productId = getReceiptFieldValue(result.receipt, 'product_id');
+	var transactionId = getReceiptFieldValue(result.receipt, 'transaction_id');
+
+	/* jshint camelcase:false */
+	if (result.hasOwnProperty('latest_receipt_info') && result.latest_receipt_info[0]) {
+		latestReceiptInfo = result.latest_receipt_info.sort(function (a, b) {
+			return parseInt(a.transaction_id, 10)  > parseInt(b.transaction_id, 10);
```

Comment:
`sort` expects a numerical return value, where you're returning a boolean.

You could however easily address by replacing `>` with `-`.